learn/wavemaker-release-notes/v11-4-0.md

```diff
@@ -56,27 +60,54 @@ Live Form in React Native is currently in beta and can be accessed upon request
 
 ## Enhancements
 
+### React Native
+
 - Added language-specific behavior like font that can be specified inside language-specific app classes. For example, use the CSS class `.app-{localecode}` to mention different font families.
 - Added an indication to show the selected option in the Select widget.
+
+### Backend
+
 - Refactored the security files in the project, by moving the spring beans related to security from XML configuration i.e. project-security.xml, which was generated in the studio design time to java configuration in the runtime.
-- Added support to include or exclude specific URLs.
-- Included ‘generated-rn-app/**', 'generated-angular-app/**’ files while exporting a project as zip.
-- Added Subtitle attribute in the Wizard widget.
+- Added support to include or exclude specific URLs for proxying REST service calls.
+- Included **generated-rn-app**, **generated-angular-app** folders while exporting the project as a zip.
+
+### UI and Accessibility
+
+- Added support to configure Subtitle attribute in Wizard widget.
 - Added Checkbox and Radio button to the Tree widget.
```

Comment:
@RitupL : Added template layout options with checkbox and Radio button to the tree widget